docker/nginx.conf

```diff
@@ -10,9 +10,9 @@ events { worker_connections 1024; }
 
 http {
 
-	# logging
-    access_log /var/log/nginx/access.log;
-    error_log /var/log/nginx/error.log;
```

Comment:
@kostajh AFAIK these paths are already symlinked to /dev/stdout in the container